app/src/css/vendor/bootstrap/_components.scss

```diff
@@ -7,12 +7,14 @@ and behaviors for our Cumulus Dashboard components
 */
 
 
-@import "~bootstrap/scss/_alert";
-@import "~bootstrap/scss/_card";
-@import "~bootstrap/scss/_close";
-@import "~bootstrap/scss/_nav";
-@import "~bootstrap/scss/_popover";
-@import "~bootstrap/scss/_progress";
-@import "~bootstrap/scss/_spinners";
-@import "~bootstrap/scss/_tooltip";
-@import "~bootstrap/scss/_transitions";
```

Comment:
Is `~` not working?

Reply:
Is end of file `new line` not allowed in .scss?

Reply:
@jjmccoy  ??